tests/Task.test.ts

```diff
@@ -194,6 +194,33 @@ describe('parsing', () => {
             });
         }
     });
+
+    it('supports parsing of tasks inside blockquotes or callouts', () => {
+        // Arrange
+        const lines = [
+            '> - [ ] Task inside a blockquote or callout 📅2022-07-29',
+            '>>> - [ ] Task inside a blockquote or callout 📅2022-07-29',
+            '> > > - [ ] Task inside a blockquote or callout 📅2022-07-29',
```

Comment:
Picking a line at random to make this observation...
I don't know if the docs mention admonitions or block quotes (I don't think they mention call-outs, as too new)... But...

I think it might be worth saying in the documentation that Tasks in call-outs and block quotes are parsed and edited fine, but not admonitions.

Again, can be done separately if you prefer.

Reply:
I do not want to couple this improvement with re-organising the documentation. I do not have the capacity to deal with the latter right now, and don't want that to block this.

I think any info arising from this PR would be fine in the following page for now:
https://obsidian-tasks-group.github.io/obsidian-tasks/getting-started/